src/oneelement.jl

```diff
@@ -427,8 +427,10 @@ function Base.reshape(A::OneElement, shape::Tuple{Vararg{Int}})
 end
 
 #permute
-
+_permute(x::Tuple, p::Tuple) = ntuple(i -> x[p[i]], length(x))
 permutedims(o::OneElementMatrix) = OneElement(o.val, reverse(o.ind), reverse(o.axes))
+permutedims(o::OneElementVector) = OneElement(o.val, (1, o.ind[1]), (1, length(o)))
+permutedims(o::OneElement, dims::Tuple) = OneElement(o.val, _permute(o.ind, dims), _permute(o.axes, dims))
```

Comment:
Do we need `dims` to be a `Tuple`? I think `AbstractVector`s would also work? Perhaps we may just have `permutedims(o::OneElement, dims)`?

Reply:
Can you add a test for the case where `dims` is not a `Tuple`?